worker.py

```diff
@@ -915,16 +1039,22 @@ def __admin_menu(self):
                                   reply_markup=telegram.ReplyKeyboardMarkup(keyboard, one_time_keyboard=True))
             # Wait for a reply from the user
             selection = self.__wait_for_specific_message([self.loc.get("menu_products"),
+			                                              self.loc.get("menu_categories"),
```

Comment:
Let's make that even with the other lines.

```suggestion
			                                        self.loc.get("menu_categories"),
```